README.md

```diff
@@ -232,14 +199,11 @@ There is a better way:
 ```ruby
 require 'hanami/model'
 
-class ArticleRepository
-  include Hanami::Repository
-
-  def self.most_recent_by_author(author, limit = 8)
-    query do
-      where(author_id: author.id).
-        order(:published_at)
-    end.limit(limit)
+class ArticleRepository < Hanami::Repository
```

Comment:
The idea in ROM is that using inheritance indicates that the class should have one responsibility, i.e. we're inheriting repository, so this class should ONLY act as a repository, discouraging people from expanding the interface with mixins